README.md

```diff
@@ -81,9 +99,23 @@ Several parameters can be specified using environment variables:
 | `zookeeper_watcher_status`        | `gauge`   | Status of watcher: -1 = NOT_STARTED, 0 = OK, 1 = WARNING, 2 = ERROR |
 | `zookeeper_write_failure_count`   | `counter` | Number of failures writing messages               |
 
+### InfluxDB Watcher
+
+| Metric                            | Type      | Description                                       |
+|-----------------------------------|-----------|---------------------------------------------------|
+| `influxdb_average_round_trip_time` | `gauge` | Average Round Trip Time in seconds |
+| `influxdb_dropped_message_count` | `counter` | Number of messages that were dropped              |
+| `influxdb_max_round_trip_time`   | `gauge`   | Maximum Round Trip Time in seconds                |
+| `influxdb_min_round_trip_time`   | `gauge`   | Minimum Round Trip Time in seconds                |
+| `influxdb_read_failure_count`    | `counter` | Number of failures reading messages               |
+| `influxdb_running_average_round_trip_time` | `gauge` | Running Average Round Trip Time in seconds for last 5 messages |
```

Comment:
cool average on client side :) 👍

Reply:
I wish I could take credit, but really all I am doing is plugging into the existing watcher interface.